src/pudl/glue/ferc1_eia.py

```diff
@@ -76,406 +78,230 @@ def get_plant_map() -> pd.DataFrame:
+        ).transform(
+            raw_dbf=ferc1_dbf_raw_dfs[table],
+            raw_xbrl_instant=ferc1_xbrl_raw_dfs[table].get("instant", pd.DataFrame()),
+            raw_xbrl_duration=ferc1_xbrl_raw_dfs[table].get("duration", pd.DataFrame()),
         )
-        .pipe(pudl.helpers.simplify_strings, ["utility_name_ferc1"])
-        .set_index(["utility_id_ferc1", "utility_name_ferc1"])
-    )
-    mapped_utils_ferc1 = (
-        get_mapped_utils_ferc1()
-        .pipe(pudl.helpers.simplify_strings, ["utility_name_ferc1"])
-        .set_index(["utility_id_ferc1", "utility_name_ferc1"])
-    )
-    unmapped_utils_ferc1 = (
-        all_utils_ferc1.loc[all_utils_ferc1.index.difference(mapped_utils_ferc1.index)]
-        .reset_index()
-        .loc[:, ["utility_id_ferc1", "utility_name_ferc1"]]
+        plant_dfs.append(plant_df)
+
+    all_plants = (
+        pd.concat(plant_dfs)
+        .drop_duplicates(["utility_id_ferc1", "plant_name_ferc1"])
+        .sort_values(["utility_id_ferc1", "plant_name_ferc1"])
     )
-    return unmapped_utils_ferc1
+    return all_plants
 
 
-def get_db_plants_eia(pudl_engine):
-    """Get a list of all EIA plants appearing in the PUDL DB.
+###############################
+# Unmapped plants and utilities
+###############################
 
-    This list of plants is used to determine which plants need to be added to
-    the FERC 1 / EIA plant mappings, where we assign PUDL Plant IDs. Unless a
-    new year's worth of data has been added to the PUDL DB, but the plants
-    have not yet been mapped, all plants in the PUDL DB should also appear in
-    the plant mappings. It only makes sense to run this with a connection to a
-    PUDL DB that has all the EIA data in it.
+
+def get_missing_ids(
+    ids_left: pd.DataFrame,
+    ids_right: pd.DataFrame,
+    id_cols: list[str],
+) -> pd.DataFrame:
+    """Identify IDs that are missing from the left df but show up in the right df.
 
     Args:
-        pudl_engine (sqlalchemy.engine.Engine): A database connection
-            engine for connecting to a PUDL SQLite database.
+        ids_left: table which contains ``id_cols`` to be used as left table in join.
+        ids_right: table which contains ``id_cols`` to be used as left table in join.
+        id_cols: list of ID column(s)
 
-    Returns:
-        pandas.DataFrame: A DataFrame with plant_id_eia, plant_name_eia, and
-        state columns, for addition to the FERC 1 / EIA plant mappings.
+    Return:
+        dataframe of unique values in ``id_cols`` that exist in ``ids_right`` but not
+        ``ids_left``.
     """
-    db_plants_eia = (
-        pd.read_sql("plants_entity_eia", pudl_engine)
-        .loc[:, ["plant_id_eia", "plant_name_eia", "state"]]
-        .pipe(pudl.helpers.simplify_strings, columns=["plant_name_eia"])
-        .astype({"plant_id_eia": int})
-        .drop_duplicates("plant_id_eia")
-        .sort_values("plant_id_eia")
+    id_test = pd.merge(
```

Comment:
I think another way to do this without needing to use all the various merge options would be to set the indexes of both dataframes to be `id_cols` and use do `ids_right.index.difference(ids_left.index)`

Reply:
using the `index.difference` would result in id's that are missing in both directions right? like if `ids_right`'s `id_cols` were `[1,2,3,4]` and `ids_left` was `[2,3,4,5]` the result would be `[1,5]` right?

oh no i am wrong!
![image](https://user-images.githubusercontent.com/25487998/194372922-a290c99f-e1bc-4667-8407-0a01c06bcc75.png)
cool yeah i think this will make this simpler